website/src/app/about/team.tsx

```diff
@@ -110,9 +110,9 @@ export default function Team() {
       ),
     },
     {
-      name: "Trisha",
+      name: "ReactorScram",
```

Comment:
It would be cool to not have my name here either

Reply:
That's fine - as long as it's consistent and we have some kind of avatar image to use. Maybe dall•e can make one?

Reply:
I found some free avatar site and made this cartoon that resembles me https://github.com/firezone/firezone/blob/abc7a92b1838d41279540e1e6d699d9d8d553fd2/website/public/images/avatars/reactorscram.png

I tried to use Dall-E from ChatGPT as recommended but it just glitched out and then said we're out of ChatGPT-4o credits or something 
<img width="857" alt="image" src="https://github.com/firezone/firezone/assets/13400041/2e5b30a1-0a77-4340-82f9-b9de2ee5f8bd">

Reply:
I could put that on GitHub too, to replace Leela